src/main/plugin-type.ts

```diff
@@ -20,6 +20,7 @@ export enum PluginType {
     ColorConverter = "color-converter",
     Dictionary = "dictionary",
     BrowserBookmarks = "browser-bookmarks",
+    BrowserHistories = "browser-histories",
```

Comment:
Why don't we call this feature `BrowserHistory`? Makes more sense in my opinion

Reply:
Thank you for your review!
I think so too, 'history' refers to `records`, and 'historyentry' refers to one record.

I'll fix these problems and refine some codes and features!